.github/workflows/continuous-integration-workflow.yml

```diff
@@ -73,6 +73,11 @@ jobs:
             cmake_build_type: 'RelWithDebInfo'
             backend: 'THREADS'
             stdcxx: '23'
+          - distro: 'fedora:latest'
+            cxx: 'g++'
+            cmake_build_type: 'RelWithDebInfo'
+            backend: 'HPX'
+            stdcxx: '17'
     runs-on: ubuntu-latest
```

Comment:
@masterleinad should that not be fedora-latest as above?

Reply:
No, we can use containers with other distros from ubuntu-latest.

Reply:
`runs-on:` is the GitHub Action base vm, which really doesn't matter much when used with `container:`.